lib/helper/text.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fns = {};
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).